ceph.spec.in

```diff
@@ -2289,9 +2284,9 @@ fi
 
 %if %{with jaeger}
 %files -n libjaeger
-%{_libdir}/libopentracing.so.*
-%{_libdir}/libthrift.so.*
-%{_libdir}/libjaegertracing.so.*
+%{buildroot}%{_libdir}/libopentracing.so.*
+%{buildroot}%{_libdir}/libthrift.so.*
+%{buildroot}%{_libdir}/libjaegertracing.so.*
```

Comment:
what is this change for?

Reply:
I am not sure, but centos is not able to include these files even if they are installed, I am testing adding %{buildroot} if it able to find it then or not
```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos8/DIST/centos8/MACHINE_SIZE/gigantic/release/17.0.0-4525-g497bb326/rpm/el8/BUILDROOT/ceph-17.0.0-4525.g497bb326.el8.x86_64
error: Installed (but unpackaged) file(s) found:
   /usr/lib64/libjaegertracing.so.0
   /usr/lib64/libjaegertracing.so.0.6.1
   /usr/lib64/libopentracing.so.1
   /usr/lib64/libopentracing.so.1.6.0
   /usr/lib64/libthrift.so.0.13.0

```
https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos8,DIST=centos8,MACHINE_SIZE=gigantic/55724//consoleFull